File: src/TrueOrFalse.Frontend.Web.Nuxt/components/question/license/licenseQuestion.ts
const DefaultLicenseId = -1

export class LicenseQuestion {
    id: number = -1
    nameLong: string = ''
    nameShort: string = ''
    displayTextShort:  string = ''
    displayTextFull: string = ''

    licenseLink: string = ''
    licenseShortDescriptionLink: string = ''

    authorRequired: boolean = false
    licenseLinkRequired: boolean = false
    changesNotAllowed: boolean = true

    IsDefault():boolean {
        return this.id == DefaultLicenseId
    }
}

File: src/Backend.Core/Domain/Question/Question.License/LicenseQuestion.cs
public class LicenseQuestion
{
    public int Id;

    public string NameLong;

    public string NameShort;

    public string DisplayTextShort;
    public string DisplayTextFull;

    public string LicenseLink;

    public string LicenseShortDescriptionLink;

    public bool? AuthorRequired;
    public bool? LicenseLinkRequired;
    public bool? ChangesNotAllowed;

    public bool IsDefault()
    {
        return Id == LicenseQuestionRepo.DefaultLicenseId;
    }
}
